src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/test_resource.py

```diff
@@ -498,8 +498,8 @@ def body(self):
 class PolicyScenarioTest(ScenarioTest):
     @ResourceGroupPreparer(name_prefix='cli_test_policy')
     def test_resource_policy(self, resource_group):
-        policy_name = 'azure-cli-test-policy'
-        policy_display_name = 'test_policy_123'
+        policy_name = self.create_random_name('azure-cli-test-policy', 30)
+        policy_display_name = self.create_random_name('test_policy', 20)
```

Comment:
could you re-record the test to ensure it can run live?

Reply:
Done